swapi/src/app/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, inject } from '@angular/core';
+import {
+  FormBuilder,
+  Validators
+} from "@angular/forms";
+import { passwordMatchValidator } from "./utils/sign-up.utils";
+
+@Component({
+  selector: 'app-sign-up',
+  templateUrl: './sign-up.component.html',
+  styleUrls: ['./sign-up.component.scss']
+})
+export class SignUpComponent {
+  private fb = inject(FormBuilder);
+
+  signUpForm = this.fb.group({
+    email: ['', {validators: [Validators.required, Validators.email]}],
+    password: ['', [Validators.required, Validators.minLength(6)]],
+    confirmPassword: ['', [Validators.required]],
```

Comment:
why no minLength here ?

Reply:
i guess it's not crucial to check `confirmPassword` minLength. However `password` and `confirmPassword` match is required

Reply:
it is wrong to have the same fields with a different set of validators

Reply:
done